lib/travis/api/v3/queries/organization.rb

```diff
@@ -1,6 +1,6 @@
 module Travis::API::V3
   class Queries::Organization < Query
-    params :id, :login, :github_id
+    params :id, :login, :github_id, :github_installation_id
```

Comment:
This should no longer be necessary. With the changes, the owner should be retrieved by fetching the installation first (ie, thus using the installation query).